Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM ubuntu:bionic
+WORKDIR /root
+
+# Install bitcoind
+RUN apt-get update \
+  && apt-get install -y software-properties-common \
+  && add-apt-repository ppa:bitcoin/bitcoin \
```

Comment:
Why do you add the PPA if you then don't download the compiled package?

Reply:
It contains libdb4.8-dev package, which is a dependency for building bitcoind.